README.md

```diff
@@ -2,3 +2,47 @@
 
 Python Munsell provides a MunsellColor object for Python.  The MunsellColor
 object can be used to represent munsell colors and convert them to RGB.
+
+
+## Dictionary Structure ##
```

Comment:
I'm not 100% sure; but I would prefer if this was inline in the `table.py` file; so that it is more obvious what the relevant dictionary is.

Reply:
When you move it there, maybe just put a paragraph in the README saying "there are docstrings in the __init__.py file (providing an abstraction for dealing with munesll colors) and table.py file (providing raw data about munsell colors"